tests/acceptance/pageObjects/FilesPageElement/versionsDialog.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  commands: {
+    /**
+     * @returns {*}
+     */
+    openVersionsSidebar: function () {
```

Comment:
the versions tab cannot open itself, should be a part of the side-bar page object, also I would propose `openVersionsTab` as name

Reply:
done!